linux/datadog_wrapper

```diff
@@ -66,9 +113,6 @@ setUpNodeEnv() {
 
 setUpDotnetEnv() {
     echo "Setting up Datadog tracing for .NET"
-
-    setUpCommonEnv || return
```

Comment:
Because the common elements have been pulled out of this method, we are no longer skipping the language specific setup when the common setup fails. I would recommend reverting this specific change (continue to rename it to `getBinaries` if you want)

Reply:
Good catch